scripts/integration-test.sh

```diff
@@ -166,7 +166,7 @@ start_node tests/node.ts \
     --bootstrapPort ${charly_port} \
     --bootstrapIdentityName 'charly' \
     --noDirectConnections true \
-    --noWebRTCUpgrade true \
+    --noWebRTCUpgrade false \
```

Comment:
I think we can also leave this option unconfigured, reduces the loc

Reply:
I prefer to be explicit in the tests so we don't hit some undefined behavior later when modifying tests